### DISCUSSION TITLE
Halt System

### DISCUSSION THREAD

**Discussion by bsbishop** [travisghansen/hass-pfsense#134] [Q&A]
I'm trying to set Home Assistant up so that when a UPS goes below a certain percentage then it will shutdown pfsense (running off same UPS). 

I see there's a pfsense: Halt System service but then it looks like it's looking for an entity ID but, while I see plenty of sensor.pfsense* IDs, I don't see anything that looks like it'd actually reference the device.

What am I missing for this to work?

**Comment by travisghansen** [comment]
Any entity that belongs to the integration should work.

**Comment by bsbishop** [comment]
That was it. Thank you!
